spec/requests/security_activity_spec.rb

```diff
@@ -199,4 +318,10 @@ def expect_event_on_security_page(event)
 
     expect(response.body).to have_content(I18n.t("account.security.event.#{event.name}"))
   end
+
+  def expect_event_on_paginated_security_page(event, page_number:, count:)
+    get account_security_paginated_path(page_number: page_number)
```

Comment:
It is woth having a test for whe no page number is passed?
Is that a possible state for the user to reach?

Reply:
Users would only get into this state if they tamper with the URL, which Rails will handle with a 404 as there won't be a `/:page_number` match in the URL.

Reply:
👍🏻